tox.ini

```diff
@@ -28,9 +28,11 @@
 ; on travis in --diff mode
 ;
 [tox]
-minversion=2.4
+minversion=3.20.1
+requires=
+    virtualenv>=20.0.33
+    tox-wheel>=0.5.0
 skip_missing_interpreters=True
-toxworkdir=build/
```

Comment:
is this change required for PEP517 ?

Reply:
seems to be a bug in tox-wheel

Reply:
Is this bug reported upstream? Is ok to have the build in .tox but I think that we should have a comment with the link to the bug... in case we want to come back the a custom build dir.

Reply:
setuptools' bdist_wheel (ab)uses the `./build` dir as a build cache, which can be flakey (cache invalidation problems) however when using `pip wheel --use-pep517` it makes a fresh tmpdir and uses the `${tmpdir}/build` dir instead

https://github.com/ionelmc/tox-wheel/pull/9/files#diff-975220820dfc0998d4b34b2c84b5b94b55c8cb2b8f4172735905fd3ab5a194aaL148